codec-classes-quic/src/main/java/io/netty/handler/codec/quic/QuicStreamResetException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2025 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.quic;
+
+/**
+ * Signals a stream reset.
+ */
+public final class QuicStreamResetException extends QuicStreamException {
```

Comment:
Does this really make sense to have this hierarchy... Maybe it would be better to just have it extend `QuicException` and rename QuicStreamException to QuicStreamStoppedException ?